common/api/core-geometry.api.md

```diff
@@ -5980,8 +5981,8 @@ export class Vector3d extends XYZ {
     fractionOfProjectionToVector(target: Vector3d, defaultFraction?: number): number;
     static fromJSON(json?: XYZProps): Vector3d;
     interpolate(fraction: number, vectorB: XYAndZ, result?: Vector3d): Vector3d;
-    isParallelTo(other: Vector3d, oppositeIsParallel?: boolean, returnValueIfAnInputIsZeroLength?: boolean): boolean;
-    isPerpendicularTo(other: Vector3d, returnValueIfAnInputIsZeroLength?: boolean): boolean;
+    isParallelTo(other: Vector3d, oppositeIsParallel?: boolean, returnValueIfAnInputIsZeroLength?: boolean, options?: PerpParallelOptions_2): boolean;
```

Comment:
not sure where and why "_2" is comming from!

Reply:
because it's same function name and args for 2d and 3d?

Reply:
Right. Put the options in Geometry.ts (use `export` on the interface) and import it from both 2d and 3d files.